frontend/src/components/ProtectedRoute.tsx

```diff
@@ -4,12 +4,12 @@ import { Route, Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
 
 function ProtectedRoute(props: any) {
-  const { currentUser, component: Component, ...rest } = props;
+  const { auth, component: Component, ...rest } = props;
   return (
     <Route
       {...rest}
       render={(props) => (
-        currentUser.name
+        auth.user
```

Comment:
I thought there was an auth.isAuthenticated? Use that maybe

Reply:
yup, gonna do that